src/Cache/FileCacheManager.php

```diff
@@ -32,6 +32,8 @@
  */
```

Comment:
> Shouldn't it be tested somehow? But I am not sure it's possible to simulate process interruption...

@Wirone , I failed to do that while remaining sane. happy for contributions here, if anyone has good idea.

Reply:
Yeah, I tried to come up with test scenario for that and found it hard too 😉. The only thing that could reflect interruption, would be something like this:
- trigger `fix` command in a process with timeout, and include some dummy fixer in the ruleset, which would do a `sleep()` _for specific file_ and cause the timeout, but also would leave some mark it was done (creating file?)
- trigger another `fix` with the same config, so cache is not invalidated, but this time there wouldn't be `sleep()` and it would go to the end

Test should verify if second run contains `S` statuses for files processed before timeouted file.

Of course there should be more than 10 files processed in the first run OR write interval should be configurable 😉.

Reply:
> OR write interval should be configurable 😉.

no changes on my side:

> for tests - I would suggest to go this part when we would have the good idea for test here.

-----

> Yeah, I tried to come up with test scenario for that and found it hard too 😉.

yeah, we can craft sth like this, but maybe it's overkill...